tests/unittests/api/utils_post_data.py

```diff
@@ -34,6 +34,14 @@
         "year": 2016,
         "logo": ""
     },
+    'licence_details': {
```

Comment:
Just like `social_links` and `version`, this field is also not included in the POST API schema so I don't think it is needed to be tested here.